assets/css/stats.css

```diff
@@ -0,0 +1,142 @@
+#stats-tabs {

```

Comment:
Should this file be a git move?

Reply:
There is a minor change, but I didn't want to odd thing to happen again. I can delete the other one.

Reply:
If there are no changes in the file, i think git move should work just fine